lib/model/query/submissions.js

```diff
@@ -338,13 +338,16 @@ const _exportUnjoiner = unjoiner(Submission, Submission.Def, Submission.Xml, Sub
 // TODO: this is a terrible hack to add some logic to one of our select fields. this is
 // the /only/ place we need to do this in the entire codebase right now. so for now
 // we just use the terrible hack.
-const { raw } = require('slonik-sql-tag-raw');
-const _exportFields = raw(_exportUnjoiner.fields.sql
-  .replace(',"submissions"."userAgent" as "submissions!userAgent"', '')
-  .replace(
-    '"submission_defs"."xml" as "submission_defs!xml"',
-    '(case when submission_defs."localKey" is null then submission_defs.xml end) as "submission_defs!xml"'
-  ));
+const xmlUnlessEncrypted = {
+  name: 'xml',
+  selectSql: tableAlias => sql`(case when ${sql.identifier([tableAlias, 'localKey'])} is null then ${sql.identifier([tableAlias, 'xml'])} end)`,
+};
+const cloneFrame = (frame, fieldMutator) => ({ ...pick(['from', 'to'], frame), fields: fieldMutator(frame.fields) });
+const ExportSubmission = cloneFrame(Submission, without(['userAgent', 'localKey']));
```

Comment:
Submission doesn't have `localKey`
```suggestion
const ExportSubmission = cloneFrame(Submission, without(['userAgent']));
```